<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerador QR - Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.ico</normaloff>../img/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="img_logo">
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>290</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>290</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../img/logoLogin.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
padding: 10px;
}
QPushButton:hover{
background: #333;
color: white;
}
QPushButton{
spacing: 30px;
}
QLineEdit{
height: 40px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="entry_user">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>User</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="entry_senha">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Senha</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="check_salvar_user">
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Salvar Usu√°rio ?</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_login">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/icons8-login-arredondado-64.png</normaloff>../icons/icons8-login-arredondado-64.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_ajuda">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ajuda</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/icons8-reproduzir-youtube-50 (1).png</normaloff>../icons/icons8-reproduzir-youtube-50 (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
